recipes/biobasehttptools/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+mv AccessionToTaxId ${PREFIX}/bin/AccessionToTaxId
+mv ${PREFIX}/bin/AccessionToTaxId ${PREFIX}/bin/AccessionToTaxId-bin
+echo -e "#!/bin/bash\nexport SYSTEM_CERTIFICATE_PATH=${PREFIX}/ssl/cacert.pem\n${PREFIX}/bin/AccessionToTaxId-bin \"\$@\"\n" > ${PREFIX}/bin/AccessionToTaxId
+chmod 755 ${PREFIX}/bin/AccessionToTaxId
+mv FetchSequence ${PREFIX}/bin/FetchSequence
+mv ${PREFIX}/bin/FetchSequence ${PREFIX}/bin/FetchSequence-bin
+echo -e "#!/bin/bash\nexport SYSTEM_CERTIFICATE_PATH=${PREFIX}/ssl/cacert.pem\n${PREFIX}/bin/FetchSequence-bin \"\$@\"\n" > ${PREFIX}/bin/FetchSequence
+chmod 755 ${PREFIX}/bin/FetchSequence
+mv GeneIdToGOTerms ${PREFIX}/bin/GeneIdToGOTerms
```

Comment:
`mv GeneIdToGOTerms ${PREFIX}/bin/`

You could also put this into a bash for loop and iterate over all your binaries.

Reply:
Nice idea, I will add that right away

Reply:
Ok, loop is added